.github/styles/Microsoft/Avoid.yml

```diff
@@ -13,6 +13,5 @@ tokens:
   - application program
   - applications developer
   - as well as
-  - ask
```

Comment:
IMHO if software sends a warning or even error it needs to be sure about it. If the software (here: Vale) is not sure because it lacks context and grammar, then it should only spit these issues out when requested and with a clear marking that is _not_ warning or error.